docs/user/openstack/provider_networks.md

```diff
@@ -0,0 +1,123 @@
+# Provider Networks
+
+
+## Table of Contents
+- [Provider Networks](#provider-networks)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Common prerequisites](#common-prerequisites)
+  - [Considerations when creating provider networks](#considerations-when-creating-provider-networks)
+  - [Deploying cluster with primary interface on a provider network with IPI](#deploying-cluster-with-primary-interface-on-a-provider-network-with-ipi)
+  - [Known issues](#known-issues)
+
+
+## Introduction
+
+Provider networks map directly to an existing physical network in the data center.
+Useful network types in this category include flat (untagged), VLAN (802.1Q tagged) and VXLAN. 
+OpenShift clusters can be deployed on provider networks, so the installer won't need to create
+tenant networks nor floating IPs.
+More information can be found about provider networks terminology [here][1].
+
+Here is a basic architecture of one OCP cluster running on a provider network and another one
+on a tenant network:
+
+![OCP on a provider network](provider-network.png)
+
+
+## Common prerequisites
+
+* [Neutron service][2] is enabled and accessible through the [OpenStack Networking API][3].
+* Neutron should be configured with the [port-security and allowed-address-pairs][4] extensions so the installer can
+  add the `allowed_address_pairs` attribute to ports.
+* The Provider Network has to be created by the administrator with specific configuration (see the next section).
```

Comment:
```suggestion
* The provider network is created by the administrator with specific configuration (see the next section).
```

If there are a number of checks to make, I'd probably remove this bullet and turn the next section into a network verification and configuration section. Otherwise, you're asking the reader to skip past the end of the prereqs.

Reply:
:+1: for a new section